mmv1/products/datastream/api.yaml

```diff
@@ -369,3 +367,428 @@ objects:
             required: true
             description: |
               A free subnet for peering. (CIDR of /29)
+  - !ruby/object:Api::Resource
+    name: 'Stream'
+    base_url: "projects/{{project}}/locations/{{location}}/streams"
+    create_url: "projects/{{project}}/locations/{{location}}/streams?streamId={{stream_id}}"
+    self_link: "projects/{{project}}/locations/{{location}}/streams/{{stream_id}}"
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/datastream/docs/create-a-stream'
+      api: 'https://cloud.google.com/datastream/docs/reference/rest/v1/projects.locations.streams'
+    description: |
+      A resource representing streaming data from a source to a destination.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: streamId
+        description: |-
+          The stream identifier.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        description: |
+          The name of the location this stream is located in.
+        required: true
+        input: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: The stream's name.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: Labels.
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        required: true
+        description: Display name.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'sourceConfig'
+        required: true
+        description: |
+          Source connection profile configuration.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'sourceConnectionProfile'
+            required: true
```

Comment:
This means that at least one source config must be specified? 
https://source.corp.google.com/piper///depot/google3/google/cloud/datastream/v1/datastream_resources.proto;rcl=491333824;l=836

Reply:
call will fail if no source config object is provided